app_constants.py

```diff
@@ -0,0 +1,3 @@
+"""Constants"""
+
+QUOTES_URL = "https://quotes.toscrape.com/"
```

Comment:
Create a folder `constants` and add the constants file there.

Reply:
Done.